wicket-kendo-ui-culture/pom.xml

```diff
@@ -25,6 +25,12 @@
 			<artifactId>wicket-jquery-ui-core</artifactId>
 			<version>6.20.1-SNAPSHOT</version>
 		</dependency>
+		<dependency>
+			<groupId>com.googlecode.wicket-jquery-ui</groupId>
+			<artifactId>wicket-kendo-ui</artifactId>
+			<version>6.20.1-SNAPSHOT</version>
+			<type>bundle</type>
```

Comment:
Was the only way to get it working. When removing ```<type>bundle</type>``` and jar-type as default gets activated, I had maven dependency errors.
My knowledge about maven and modulized projects and bundles is a bit poor here. I must admit, it was trail-and-error.